### DISCUSSION TITLE
How to see Response Transformer in dashboard?

### DISCUSSION THREAD

**Discussion by daniel-ls-howard** [nextgenhealthcare/connect#5943] [Q&A] [answered]
When I view channel messages in the dashboard and look at the details, some destinations will show radio buttons including "Response Transformed" and "Processed Response".

I have a channel with a destination with a response and a response transformer but I don't see those when I look at my messages. 

In my case my destination calls a channel which returns XML then my response transformer translates that to json.

My Data Types are set as:

Source Connector inbound XML -> outbound JSON
Destination (Type JavaScript) Inbound JSON -> outbound (xml) because this channel returns XML. But my final response to the client is JSON.

Any ideas?

**Comment by tonygermano** [comment, marked as answer]
See https://docs.nextgen.com/bundle/Mirth_User_Guide_41/page/connect/connect/topics/c_Message_Storage_Settings_connect_ug.html

Your channel storage mode needs to be set to Development for it to store the content you would like to see.

**Comment by daniel-ls-howard** [comment]
Thanks! You rock! :)
